### PR TITLE
Csp 60 update readme

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,165 +1,82 @@
...
+
+Once the full installation is done (below), you can run the API by running the SFA.DAS.Roatp.Api project and you will see a web interface for all available endpoints (via swagger)
+
+
+# Jobs
+
+Once the full installation is done (below), you will need to run the SFA.DAS.Roatp.Jobs project
+
+In addition, you will need the following repo set up and running for the jobs to run:
+* das-apim-endpoints, CourseManagement solution (see https://github.com/SkillsFundingAgency/das-apim-endpoints/)
```
Comment (DevenKShah) [inline README.md:30]: Add course mgmt folder at the end of the link to navigate directly to the specific folder

File: README.md [hunk 2]
```diff
@@ -1,165 +1,82 @@
...
+
+##### Open the API solution
+
+- Open Visual studio as an administrator
+- Open the solution
+- Set SFA.DAS.RoATP.Api as the startup project
+- Publish the database project SFA.DAS.RoATP.Database to your local SQL Server instance
+- Running the solution will launch the API in your browser
+
+##### Open the Jobs solution
```
Comment (DevenKShah) [inline README.md:75]: There is no separate jobs solution, is there?

**Review by DevenKShah — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (MarkFCain) [reply README.md:75]: yeah I'll reword

File: README.md (hunk 1 above)
Comment (MarkFCain) [reply README.md:30]: will do

File: README.md [hunk 3]
```diff
@@ -1,165 +1,82 @@
...
+# ![crest](https://assets.publishing.service.gov.uk/government/assets/crests/org_crest_27px-916806dcf065e7273830577de490d5c7c42f36ddec83e907efe62086785f24fb.png) Digital Apprenticeships Service
 
-# _Project Name_
+# Register of Apprenticeship Training Providers (RoATP) Course Management API
 
 <img src="https://avatars.githubusercontent.com/u/9841374?s=200&v=4" align="right" alt="UK Government logo">
 
-_Update these badges with the correct information for this project. These give the status of the project at a glance and also sign-post developers to the appropriate resources they will need to get up and running_
 
-[![Build Status](https://dev.azure.com/sfa-gov-uk/Digital%20Apprenticeship%20Service/_apis/build/status/_projectname_?branchName=master)](https://dev.azure.com/sfa-gov-uk/Digital%20Apprenticeship%20Service/_build/latest?definitionId=_projectid_&branchName=master)
```
[the remark this answers is not in the document]
Comment (MarkFCain) [reply README.md:9]: I have added back with the correct projectId ('SkillsFundingAgency_das-roatp-api') but it's saying stuff like not computed' on the main branch.  It's computing it for branches though.  Is this correct, or is some sort of config change needed for this to present correctly.  See https://sonarcloud.io/project/overview?id=SkillsFundingAgency_das-roatp-api for an example

**Pull request merged**